src/oskari.dom.js

```diff
@@ -113,6 +113,17 @@ const isEmbedded = () => {
     return getMapContainerEl().classList.contains(APP_EMBEDDED_CLASS);
 };
 
+const showNavigation = (show) => {
+    const nav = [...Oskari.dom.getRootEl().children].find(c => c.localName === 'nav');
```

Comment:
Let's add and expose getNavigationEl() here since this line is starting to be in a lot of places in the code.